packages/explorer/src/app/worlds/[worldAddress]/utils/getFieldIndex.ts

```diff
@@ -0,0 +1,13 @@
+import { isDynamicAbiType, isStaticAbiType } from "@latticexyz/schema-type/internal";
+
+export function getFieldIndex(valueSchema: Record<string, string>, fieldName: string) {
```

Comment:
might be worth adding this into protocol-parser

Reply:
totally, created a PR https://github.com/latticexyz/mud/pull/3046